tests/integration/conftest.py

```diff
@@ -23,6 +25,22 @@
 logger = logging.getLogger(__name__)
 
 
+async def get_mattermost_ip(
+    ops_test: OpsTest,
+    app: Application,
+):
+    """Get the IP address of the leader unit of mattermost.
```

Comment:
Do you think any new functions we're adding should comply with our docstring standards etc?

Reply:
I applied our docstring standards to the new utils module